03_ROS/ghost_sim/include/ghost_sim/test_publisher_v5_actuator_cmd.hpp

```diff
@@ -33,6 +33,10 @@ class testPublisherV5ActuatorCmd : public rclcpp::Node {
 
 	void publishData();
 	rclcpp::Publisher<ghost_msgs::msg::V5ActuatorCommand>::SharedPtr output_pub_;
+	std::array<std::string, 8> motors_{"DRIVE_LEFT_FRONT_LEFT_MOTOR",
```

Comment:
This is acutally a test file that makes pseudo data as if you are sending motor cmds to the individual motors. This will not run on the robot in real time. So I think this implementation is fine for the scope of this file.